### DISCUSSION TITLE
Could not find com.bihe0832:GradleDependenciesCheck:1.0.2.

### DISCUSSION THREAD

**Discussion by WorldWarriorIII** [openTCS/opentcs#163] [Q&A]
A problem occurred configuring root project 'openTCS'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not find com.bihe0832:GradleDependenciesCheck:1.0.2.
     Searched in the following locations:
       - https://jcenter.bintray.com/com/bihe0832/GradleDependenciesCheck/1.0.2/GradleDependenciesCheck-1.0.2.pom
       - https://plugins.gradle.org/m2/com/bihe0832/GradleDependenciesCheck/1.0.2/GradleDependenciesCheck-1.0.2.pom
       - https://repo1.maven.org/maven2/com/bihe0832/GradleDependenciesCheck/1.0.2/GradleDependenciesCheck-1.0.2.pom
     Required by:
         project :

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html

**Comment by swltr** [comment]
@WorldWarriorIII Which version / Git revision? Have you made any local modifications to the source code?
